homeworks/HW10_Flask_Intro/app.py

```diff
@@ -0,0 +1,46 @@
+from flask import Flask, render_template
+
+app = Flask(__name__)
+
+
+@app.route('/', methods=['POST', 'GET'])
+def hello_world():
+    return 'Hey, it\'s a Flask calculator! ' \
+           'Please enter "/calc/"in the address bar, ' \
+           'then enter the first number, second number and the action, separated by a "/" sign. ' \
+           'Use: ' \
+           '"div" for division, ' \
+           '"sum" for addition, ' \
+           '"dif" for subtraction, ' \
+           '"mult" for multiplication.' \
+           'Thank you! :)'
+
+
+@app.route('/calc/')
+def hello_calc():
+    return "Good! Now enter numbers and action!"
+
+
+@app.route('/calc/<int:x>/<int:y>/')
+def hello_calc_(x, y):
+    return f"Good job! You entered {x} and {y}! Now enter an action!"
+
+
+@app.route('/calc/<int:x>/<int:y>/<string:action>')
+def calc(x, y, action):
+    if action == 'div':
```

Comment:
тут можна зробити дікт з екшинами. наприклад
```suggestion
    actions = { 'div': render_template('calc.html', y=y)...}
    return actions.get(action)()
```